curryreader.py

```diff
@@ -65,6 +74,8 @@ def read(inputfilename='', plotdata = 1, verbosity = 2):
             # handle cancel
             if not filepath:
                 raise Exception
+        except ImportError as e:

```

Comment:
Clearly we don't yet have pedantic CIs in place :)
```suggestion
        except ImportError:

```

Reply:
that's a good thing, given that we want to largely keep the original code :)